client/gui/CGuiHandler.h

```diff
@@ -48,33 +49,34 @@ class CGuiHandler
 	CGStatusBar * statusbar;
 
 private:
-	typedef std::list<CIntObject*> CIntObjectList;
+	typedef std::list<CIntObject *> CIntObjectList;
 
 	//active GUI elements (listening for events
 	CIntObjectList lclickable,
-				   rclickable,
-				   mclickable,
-				   hoverable,
-				   keyinterested,
-				   motioninterested,
-	               timeinterested,
-	               wheelInterested,
-	               doubleClickInterested,
-	               textInterested;
+	rclickable,

```

Comment:
This not really intended. It's problem caused by "indent_continue = -1". But other than that we talked with @alexvins and I come to conclusion it's better if we won't declare using this comma syntax.

Basically it's would be better if they were declared on their own lines.
```
	CIntObjectList lclickable;
	CIntObjectList rclickable;
	CIntObjectList mclickable;
	CIntObjectList hoverable;
	CIntObjectList keyinterested;
	CIntObjectList motioninterested;
	CIntObjectList timeinterested;
	CIntObjectList wheelInterested;
	CIntObjectList doubleClickInterested;
	CIntObjectList textInterested;
```
It's of course will be more code, but it's not as error-prone as declaration with comma especially when it's come to working with pointers.